x-pack/plugin/logsdb/src/yamlRestTest/resources/rest-api-spec/test/40_source_mode_setting.yml

```diff
@@ -436,161 +318,14 @@ create an index with standard index mode:
           settings:
             index:
               mode: standard
-          mappings:
-            _source:
-              mode: stored
-
-  - do:
-      indices.get_mapping:
-        index: test_standard_index_mode
-
-  - match: { test_standard_index_mode.mappings._source.mode: stored }
-
----
-create an index with time_series index mode and synthetic source:
```

Comment:
These tests are now repeated above, since there's only one way to configure source mode.